sklearn/tests/test_docstring_parameters.py

```diff
@@ -216,7 +216,8 @@ def test_fit_docstring_attributes(name, Estimator):
     else:
         est.fit(X, y)
 
-    skipped_attributes = {'n_features_in_'}
+    # TODO: Remove "coef_" and "intercept_" in version 0.26
+    skipped_attributes = {'n_features_in_', 'coef_', 'intercept_'}
```

Comment:
We should not skip these. We need something similar to:

https://github.com/scikit-learn/scikit-learn/blob/8cce5bfcf0f001d91deca145f878f474a0b8e894/sklearn/multiclass.py#L446-L457

make sure `hasattr` fails.

Reply:
I have made a few of changes to check that `FutureWarning` is raised for deprecated attributes when testing the docstring.